lib/crystalball/predictor/associated_specs.rb

```diff
@@ -12,9 +12,9 @@ def initialize(from:, to:)
       end
 
       def call(diff, _)
-        diff.map(&:relative_path).grep(from).map do |source_file_path|
-          to % captures(source_file_path)
-        end
+        diff.map(&:relative_path).grep(from)
+            .map { |source_file_path| to % captures(source_file_path) }
+            .select { |spec_file_path| File.exist?(spec_file_path) }
```

Comment:
I think it's better to move this check to upper level `Predictor#cases` since any of predictors can return specs which don't exist in latest masters. You'll also have to cut off `[1:1:2:3]` part

Reply:
+ we need a feature for it